src/python-common/ceph/deployment_utils/recips.py

```diff
@@ -0,0 +1,15 @@
+from ceph.mon_command_api import MonCommandApi
+
+def osd_empty(osd_id):
+    pass
+
+def remove_osd(osd_id, rados):
+    # Would this work?
+    # Does this make sense?
+    # This looks like it should live in the orchestrator.py instead?
+    api = MonCommandApi(rados)
+    api.osd_out([osd_id])
+    osd_empty(osd_id)
+    api.osd_purge(osd_id)
+    api.orchestrator_service_instance('stop', svc_type='osd', svc_id=osd_id)
+    api.orchestrator_osd_zap(osd_id)
```

Comment:
Would anyone except the SSH orchestrator make use of this?

Reply:
@BlaineEXE does this related to https://github.com/rook/rook/pull/3369 somehow? Would be nice to exchange thoughts here.

Reply:
I'm no sure what relation you're seeing. rook/rook#3369 isn't directly related to to OSDs.